server/controllers/listings/lib/elements.ts

```diff
@@ -25,11 +26,19 @@ export async function getElementsByListings (listingsIds) {
   return db.getDocsByViewKeys<ListingElement>('byListings', listingsIds)
 }
 
+export async function getElementsByListing (listingId) {
+  return db.getDocsByViewQuery<ListingElement>('byOrdinalAndListing', {
+    startkey: [ listingId, 0 ],
+    endkey: [ listingId, 'z' ],
```

Comment:
This was because uppercases were used before for defining ordinals. But since ff9afd56d max and min keys can indeed be of use now.
Done with 1ac3efff4